client/src/game/carrier.js

```diff
@@ -56,6 +58,29 @@ class Carrier extends EventEmitter {
     this.maxScale = this.userSettings.map.objectsMaximumScale/4.0
 
     Carrier.zoomLevel = userSettings.map.zoomLevels.carrierShips
+
+    this.clearPaths() // clear on setup since this is used to reset waypoints
+  }
+
+  addContainerToChunk(chunks, firstX, firstY) {
+    let chunkX = Math.floor(this.data.location.x/Map.chunkSize)
+    let chunkY = Math.floor(this.data.location.y/Map.chunkSize)
+    let ix = chunkX-firstX
+    let iy = chunkY-firstY
+
+    chunks[ix][iy].addChild(this.container)
+    chunks[ix][iy].mapObjects.push(this)
+  }
+
+  removeContainerFromChunk(chunks, firstX, firstY) {
```

Comment:
This method too. Feels wrong to reference the map chunks inside the carrier class.